doc/nrf/config_and_build/bootloaders/bootloader_external_flash.rst

```diff
@@ -13,14 +13,18 @@ When using MCUboot, you can store the storage partition for the secondary slot i
 * Single-byte read and write.
 * Writing data from the internal flash memory to the external flash memory.
 
+MCUboot typically increases slot sizes when external flash is enabled.
+Therefore you must accordingly adjust the ``CONFIG_BOOT_MAX_IMG_SECTORS`` Kconfig option, which defines the maximum number of image sectors MCUboot can handle.
+For detailed configuration instructions see `MCUboot's Kconfig options used in Zephyr <https://github.com/nrfconnect/sdk-mcuboot/blob/main/boot/zephyr/Kconfig#L370>` for details.
```

Comment:
The link looks odd in generated docs:
![image](https://github.com/nrfconnect/sdk-nrf/assets/89006246/6a177398-77c4-49e4-abbe-671fe1b9f886)

Reply:
Also, use "detail" twice in one sentence. One can be removed